tests/fsm/test_statistical.py

```diff
@@ -75,17 +63,17 @@ def prob_markov(token: List[int]) -> np.array:
 
     n_samples: int = 250
     regex_str: str = r"11[01]+|0[01]*"
-    tokenizer = MockTokenizer()
     model1 = NextToken(prob_markov, p0, states, 30127)
     model2 = NextToken(prob_non_markov, p0, states, 24601)
 
     lengths1: np.array = np.zeros((n_samples,))
     lengths2: np.array = np.zeros((n_samples,))
     for i in range(n_samples):
-        out1: List[int] = generate(model1, tokenizer, regex_str)
-        lengths1[i] = len(out1) - 1  # take off the eos token
-        out2: List[int] = generate(model2, tokenizer, regex_str)
-        lengths2[i] = len(out2) - 1  # take off the eos token
+        out1: List[int] = generate(model1, regex_str)
+        print(out1)
+        lengths1[i] = len(out1) - 1
+        out2: List[int] = generate(model2, regex_str)
+        lengths2[i] = len(out2) - 1
 
     # 2 sample KS test to check that lengths has the same distribution as
```

Comment:
The comments from here down document how the value in the assert is calculated. There is a true value, which should be recovered when n_samples is extremely large. The value in the assert is the value we get for these seeds and value of n_samples. The large sample behaviour was verified, but is too slow to run every time we want to run the tests. If these values slip then it's a sign something changed.